.github/workflows/check-licenses.yml

```diff
@@ -14,16 +14,17 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
-      - name: Install Go
-        uses: actions/setup-go@v5
-        with:
-          go-version: ${{ env.GO_VERSION }}
-
       - name: Load Go cache
         uses: ./.github/actions/go-cache
         with:
           key: lint
 
+      - name: Install Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ env.GO_VERSION }}
+          cache: false
```

Comment:
so it's the same thing I suggested  `cache=false`

Reply:
Yes, quite detailed research was done to find out why this is the right solution.